### PR TITLE
Add support for newer morpho tools

### PR DESCRIPTION
**Pull request by st4rl3ss**
Bumped a few version and fixed a couple of dependencies!

### CODE REVIEW

File: var/spack/repos/builtin/packages/morpho-kit/package.py [hunk 1]
```diff
@@ -16,9 +16,12 @@ class MorphoKit(CMakePackage):
...
 
     version('develop', branch='master', submodules=True, get_full_repo=True)
+    version('0.2.0', tag='v0.2.0', submodules=True, get_full_repo=True)
     version('0.1.0', tag='v0.1.0', submodules=True, get_full_repo=True)
 
     depends_on('cmake@3.2:', type='build')
     depends_on('py-setuptools-scm', type='build')
     depends_on('morphio', when='@0.1.1:')
     depends_on('morphio@:2.2.1', when='@0.1.0')
+    depends_on('morphio@:2.3.9', when='@0.2.0')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/morpho-kit/package.py]: Is there a reason for the upper bound?
```suggestion
    depends_on('morphio@2.3.9:', when='@0.2.0:')
```

File: var/spack/repos/builtin/packages/py-morpho-kit/package.py [hunk 2]
```diff
@@ -14,12 +14,15 @@ class PyMorphoKit(PythonPackage):
...
 
     version('develop', branch='master', submodules=True, get_full_repo=True)
+    version('0.2.0', tag='v0.2.0', submodules=True, get_full_repo=True)
     version('0.1.0', tag='v0.1.0', submodules=True, get_full_repo=True)
 
     depends_on('py-setuptools', type='build')
 
     depends_on('morphio~mpi', type=('build', 'link'), when='@0.1.0:')
     depends_on('morphio~mpi@:2.2.1', type=('build', 'link'), when='@0.1.0')
+    depends_on('morphio~mpi@:2.3.9', type=('build', 'link'), when='@0.2.0')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-morpho-kit/package.py]: See above :)
```suggestion
    depends_on('morphio~mpi@2.3.9:', type=('build', 'link'), when='@0.2.0:')
```

File: var/spack/repos/builtin/packages/py-morpho-kit/package.py [hunk 3]
```diff
@@ -14,12 +14,15 @@ class PyMorphoKit(PythonPackage):
...
 
     depends_on('cmake@3.2:', type='build')
     depends_on('py-numpy', type='run')
+    depends_on('boost')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-morpho-kit/package.py]:
```suggestion
    depends_on('boost', when='@0.2.0:')
```

File: var/spack/repos/builtin/packages/morpho-kit/package.py [hunk 4]
```diff
@@ -16,9 +16,12 @@ class MorphoKit(CMakePackage):
...
+    depends_on('boost')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/morpho-kit/package.py]:
```suggestion
    depends_on('boost', when='@0.2.0:')
```

**Review by matz-e — COMMENTED** [review]
Minor details :)

**Review by alkino — APPROVED** [review]

**Review by wizmer — APPROVED** [review]

**Comment by pramodk** [comment]
@st4rl3ss : just `a nitpick` : when we squash & merge, you can also update commit message with original, small summary. Otherwise by default we get something like:

```

    Add support for newer morpho tools (#819)

    * Add support for newer morpho tools

    * Fixes small typo

    * Update var/spack/repos/builtin/packages/morpho-kit/package.py

    Co-authored-by: Matthias Wolf <m+git@sushinara.net>

    * Update var/spack/repos/builtin/packages/py-morpho-kit/package.py

    Co-authored-by: Matthias Wolf <m+git@sushinara.net>

    * Update var/spack/repos/builtin/packages/py-morpho-kit/package.py

    Co-authored-by: Matthias Wolf <m+git@sushinara.net>

    * Update var/spack/repos/builtin/packages/morpho-kit/package.py

    Co-authored-by: Matthias Wolf <m+git@sushinara.net>

    * Removed unnecessary import

    Co-authored-by: Matthias Wolf <m+git@sushinara.net>
```

**Comment by st4rl3ss** [comment]
> @st4rl3ss : just `a nitpick` : when we squash & merge, you can also update commit message with original, small summary. Otherwise by default we get something like:
> 
> ```
> 
>     Add support for newer morpho tools (#819)
> 
>     * Add support for newer morpho tools
> 
>     * Fixes small typo
> 
>     * Update var/spack/repos/builtin/packages/morpho-kit/package.py
> 
>     Co-authored-by: Matthias Wolf <m+git@sushinara.net>
> 
>     * Update var/spack/repos/builtin/packages/py-morpho-kit/package.py
> 
>     Co-authored-by: Matthias Wolf <m+git@sushinara.net>
> 
>     * Update var/spack/repos/builtin/packages/py-morpho-kit/package.py
> 
>     Co-authored-by: Matthias Wolf <m+git@sushinara.net>
> 
>     * Update var/spack/repos/builtin/packages/morpho-kit/package.py
> 
>     Co-authored-by: Matthias Wolf <m+git@sushinara.net>
> 
>     * Removed unnecessary import
> 
>     Co-authored-by: Matthias Wolf <m+git@sushinara.net>
> ```

Sorry... I missed that this time.

**Pull request merged**